nvme.c

```diff
@@ -7030,15 +7037,22 @@ static int write_zeroes(int argc, char **argv, struct command *cmd, struct plugi
 		.storage_tag	= cfg.storage_tag,
 		.dspec		= cfg.dspec,
 		.timeout	= nvme_cfg.timeout,
-		.result		= NULL,
+		.result		= &result,
 	};
 	err = nvme_write_zeros(&args);
 	if (err < 0)
 		nvme_show_error("write-zeroes: %s", nvme_strerror(errno));
 	else if (err != 0)
 		nvme_show_status(err);
-	else
+	else {
 		printf("NVME Write Zeroes Success\n");
+		if (cfg.nsz) {
+			if (result & 0x1)
+				printf("All logical blocks in the entire namespace cleared to zero\n");
+			else
+				printf("%d logical blocks cleared to zero\n", cfg.block_count);
+		}
+	}
```

Comment:
I think it would be better to be silent on success and print the success message only if the `--verbose` flag is provided. So the usual unix tool behavior. Rest looks good.

Reply:
@igaw Addressed the review comment.
Thanks!